behaviours/meme_rater.py

```diff
@@ -125,6 +138,8 @@ async def rate_meme(message: hikari.Message, ratings: list[int]) -> None:
             10,
         )
 
+        str_explanations = "                   ".join(explanations)
```

Comment:
Can you wrap the explanations over new lines. This looks ass.

Why does it render as a tuple too in your screenshots. Please fix that or attach updated screenshots.